### DISCUSSION TITLE
bootstrap got errors, where to found the logs?

### DISCUSSION THREAD

**Discussion by dddinary** [LazyVim/LazyVim#226] [Q&A] [answered]
When the LazyVim starts, I got many error notices:
![image](https://user-images.githubusercontent.com/26164978/217999220-ca110073-3116-49a0-8e9a-dbf90c3a5861.png)
It looks like some plugins are failed to load, but I didn't change any config file, What should I do?
Thanks♪(･ω･)ﾉ

**Comment by folke** [comment, marked as answer]
You probably had errors during installation? Spotty internet? There's likely empty directories for plugins on your system. Delete those and open Neovim again. To nmake it work on spotty internet connection, you should set `lua/config/lazy.lua` the `config.concurrency = 10` or something like that

**Reply by dddinary** [reply]
It works for me! THANKS!
